### PR TITLE
[TwigBundle][TwigBridge] Improving deprecation message of the legacy Twig template directory

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | 4.2
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | yes
| Tests pass?   | yes
| Fixed tickets | #30088 
| License       | MIT

#eu-fossa

### CODE REVIEW

**Comment by yceruto** [comment]
Updating tests

File: src/Symfony/Bridge/Twig/Command/DebugCommand.php [hunk 1]
```diff
@@ -379,7 +379,7 @@ private function findWrongBundleOverrides(): array
                     $path = ltrim($relativePath.$name, \DIRECTORY_SEPARATOR);
                     $carry[$name] = $path;
 
-                    @trigger_error(sprintf('Templates directory "%s" is deprecated since Symfony 4.2, use "%s" instead.', $absolutePath, $this->twigDefaultPath.'/bundles/'.$name), E_USER_DEPRECATED);
+                    @trigger_error(sprintf('Loading Twig templates from "%s" directory is deprecated since Symfony 4.2, use "%s" instead.', $absolutePath, $this->twigDefaultPath.'/bundles/'.$name), E_USER_DEPRECATED);
```
Comment (xabbuh) [inline src/Symfony/Bridge/Twig/Command/DebugCommand.php]: [...] from the "%s" [...]

File: src/Symfony/Bridge/Twig/Tests/Command/DebugCommandTest.php [hunk 2]
```diff
@@ -65,7 +65,7 @@ public function testWarningsWrongBundleOverriding()
 
     /**
      * @group legacy
-     * @expectedDeprecation Templates directory "%sResources/BarBundle/views" is deprecated since Symfony 4.2, use "%stemplates/bundles/BarBundle" instead.
+     * @expectedDeprecation Loading Twig templates from "%sResources/BarBundle/views" directory is deprecated since Symfony 4.2, use "%stemplates/bundles/BarBundle" instead.
```
Comment (xabbuh) [inline src/Symfony/Bridge/Twig/Tests/Command/DebugCommandTest.php]: [...] from the "%sResources/BarBundle/views" [...]

File: src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php [hunk 3]
```diff
@@ -145,7 +145,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $viewsPaths = [];
         if (is_dir($dir = $rootDir.'/Resources/views')) {
             if ($dir !== $this->defaultViewsPath) {
-                $notice = sprintf('Storing templates in the "%s" directory is deprecated since Symfony 4.2, ', $dir);
+                $notice = sprintf('Loading Twig templates from "%s" directory is deprecated since Symfony 4.2, ', $dir);
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php]: [...] from the "%s" [...]

**Comment by yceruto** [comment]
Comments addressed, thank you @xabbuh

**Review by fabpot — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @yceruto.

**Pull request merged**